tower-grpc-build/src/lib.rs

```diff
@@ -22,10 +22,17 @@ struct Inner {
     build_server: bool,
 }
 
+struct GeneratorState {
+    root: codegen::Scope,
+    // XXX this is where it gets _really_ ugly.
```

Comment:
is this comment actionable? should something change? if not, please remove.

Reply:
some upstream changes were recently made in `prost-build` (danburkert/prost#65) which will let us make that code much less ugly (see discussion on PR #10), so this can hopefully go away soon.

Reply:
I'd prefer that we take out anything like "this is ugly" or "this is a terrible hack" and replace it with something actionable.

Reply:
I've changed this comment in the branch that this branch should logically be based on (#10); when I rebase this branch onto that branch, this comment will be replaced. It now looks like this:
https://github.com/tower-rs/tower-grpc/blob/c7572dd92dda90c5bc2f484c41b323a1557834d4/tower-grpc-build/src/lib.rs#L32-L35

Is that better?